README.md

```diff
@@ -28,30 +29,41 @@ To install vCluster Rancher Operator, you can add the loft repository to Rancher
+
+*Downstream Client*
+
+The vCluster Rancher operator runs in Rancher's local cluster and watches clusters.management.cattle.io. 
+When a cluster has not been seen by the handler before, it creates a client using the Rancher reverse proxy to talk to the downstream cluster. This is done by creating a token, if needed, for the user.management.cattle.io resource
 that is created during the Helm chart install. The token is then used to authenticate requests to the Rancher proxy.
 
-Once a client is created for a downstream cluster, it is used to watch all services with the label `app=vcluster`. If the vCluster install does not have a corresponding
-Rancher cluster, the installation process begins. The installation process creates a provisioning clusters, waits for Rancher to create the clusterregistrationtoken.cattle.io,
-and then extracts the command from the clusterregistrationtokens status. This command contains everything that is needed to deploy Rancher's cluster agent to the vCluster. A
-job is then deployed in the vCluster's host cluster that creates a kubeconfig pointing the the vCluster service in the host cluster. The earlier extracted command is executed
-using the newly created kubeconfig.
+*Rancher Cluster Creation*
 
-The service event handler then figures out which project the vCluster is installed in. All Rancher users that have a projectroletemplatebidning.cattle.io for the project with either the
-project-owner or project-member roles are added as cluster owners, using a clusterroletemplatebinding.management.cattle.io, to the vCluster Rancher cluster.
+Once a client is created for each downstream cluster, it is used to watch all services with the label `app=vcluster`. If a virtual cluster install is found which does not have a corresponding Rancher cluster, the installation process begins. 
+The installation process creates a provisioning cluster, waits for Rancher to create the `clusterregistrationtoken.cattle.io`,
+and then extracts the command from the `clusterregistrationtoken` status. This command contains everything that is needed to deploy Rancher's cluster agent to the vCluster.
+A job is then deployed in the vCluster's host cluster that creates a kubeconfig pointing to the vCluster service in the host cluster. The earlier extracted command is executed using the newly created kubeconfig.
 
 Deletion events for the vCluster service will trigger the controller to delete the corresponding Rancher cluster.
 
+*Project Management*
+
+The service event handler then figures out which project the vCluster is installed in. All Rancher users that have a `projectroletemplatebidning.cattle.io` for the project with either the `project-owner` or `project-member` roles are added as cluster owners, using a `clusterroletemplatebinding.management.cattle.io`, to the virtual cluster's Rancher cluster.
```

Comment:
This is now handled by the clusters.management.cattle.io handler. Creating the provisioning cluster creates a clusters.management.cattle.io, which then triggers the handler for the new cluster. Any management cluster with vcluster labels will go through a RBAC syncing process. This handler acts on clusters.management.cattle.io but also watches projectroletemplatebindings.cattle.io. The watcher will checks if any vCluster Rancher clusters are associated with the project targeted by a prtb referenced in a prtb event and enqueues those clusters if so.